roles/lib_openshift/molecule/origin-latest/tests/test_oc_secret.py

```diff
@@ -0,0 +1,157 @@
+import pytest
```

Comment:
pytest is brought in because of molecule, which uses testinfra as the default testing backend and is a plugin for pytest.

I do like the idea of changing to use pytest instead of nose for the test runner, but iirc we would need to refactor some of the existing tests and I didn't want to go down that road *yet*